tests/test_dpfresultwithkeywords.py

```diff
@@ -306,11 +388,36 @@ def test_groupingmat_elemlocation(allkindofcomplexity):
     stress = s.tensor
     assert stress.num_fields == 11
     assert len(stress[1]) == 9828
-    assert len(stress[5]) == 156762
-    assert stress.get_data_at_field(5)[0][2] == 2255995657.8043337
+    assert len(stress[5]) == 52254
+    assert stress.get_data_at_field(5)[0][2] == 2089125611.1128974
```

Comment:
Should be using `np.isclose`.  There might be floating point round off errors and those shouldn't cause the test to fail.